dev_guide/builds/build_strategies.adoc

```diff
@@ -525,3 +525,48 @@ spec:
     file to use, relative to the source `contextDir`.
     If `contextDir` is omitted, it defaults to the root of the repository.
     If `jenkinsfilePath` is omitted, it defaults to *_Jenkinsfile_*.
+
+[[jenkins-pipeline-strategy-environment]]
+=== Environment Variables
+
+To make environment variables available to the
+xref:../../architecture/core_concepts/builds_and_image_streams.adoc#pipeline-build[Pipeline build]
+process, you can add environment variables to the `jenkinsPipelineStrategy` definition
+of the `BuildConfig`.
+
+The environment variables defined there are set as parameters on the job in Jenkins associated with
+the `BuildConfig`. 
+
+For example:
```

Comment:
The above is a little repetitive. Can I suggest it be merged? So:

Add environment variables to the `jenkinsPipelineStrategy` section of a `BuildConfig` to make environment variables available to a xref:../../architecture/core_concepts/builds_and_image_streams.adoc#pipeline-build[Pipeline build]. Once defined, the environment variables will be set as parameters for any Jenkins job associated with the `BuildConfig`.